### ISSUE TITLE
[Question]: can we use "rekor.tlog.dev" address as a value of the annotation "chains.tekton.dev/transparency"?

### ISSUE THREAD

**Issue by developer-guy** [tektoncd/chains#576]
### Feature request

In the meantime, Tekton Chains adds a transparency log server API URL which is "rekor.sigstore.dev"_ to the annotation _"chains.tekton.dev/transparency"_ [^1] but the Chainguard team developed a UI project for Rekor called [rekor-search-ui](https://github.com/chainguard-dev/rekor-search-ui), so, I thought that should be a better experience for the end-users to change this URL with the UI one?

_rekor.sigstore.dev => rekor.tlog.dev_

[^1]: https://github.com/tektoncd/chains/blob/b34353430a408dc32dac20d29e1d9da948380996/pkg/chains/signing.go#L238

**Comment by developer-guy** [comment]
>I prefer the rekor.sigstore.dev URL so that people can easily know which tlog the entry is in.
Yeah, that makes sense to me either, but from the end-user perspective, when they display the details of the TaskRun, IMHO, it'd be better to see the UI link (https://rekor.tlog.dev/?logIndex=735223) instead of the API link because UI's are a super cool feature for the end-users 🙈

**Comment by developer-guy** [comment]
any other thoughts?

**Comment by imjasonh** [comment]
rekor.tlog.dev has no SLO and may be down or deleted at any time without warning.

The rekor.sigstore.dev annotation key is named like a URL not because it should be resolved in a browser, but just to namespace it. 

The HTML served at rekor.sigstore.dev can presumably also go away at any time.

I recommend against using tlog.dev for this.

**Comment by wlynch** [comment]
+1 to Jason / Priya w.r.t. the existing value.

If we *really* want this maybe we could put this in a separate configurable annotation? 🤔

**Comment by developer-guy** [comment]
>If we really want this maybe we could put this in a separate configurable annotation? 🤔

This would be great if we could! I'm so thrilled to do that 🙈

**Comment by developer-guy** [comment]
based on the recent news that Chainguard contributes Rekor Search UI project to Sigstore, we might reconsider using this address in one of the Tekton Chains annotations about the transparency log server. WDYT @wlynch @imjasonh?

https://www.chainguard.dev/unchained/chainguard-contributes-rekor-search-project-to-sigstore
